### PR TITLE
A star sends an invite

### PR DESCRIPTION
**Pull request by willbach**

### CODE REVIEW

File: src/components/L2/Card.tsx
Comment (tomholford) [inline src/components/L2/Card.tsx:29]: Functionally it shouldn't make a difference, but given the expected values defined by the `CardProps` it appears that `!!` is extraneous here. This is certainly a nitpick, so optional to act on it, but a change would improve the readability of this section.

File: src/components/L2/Dropdowns/AccountsDropdown.tsx
Comment (tomholford) [inline src/components/L2/Dropdowns/AccountsDropdown.tsx:34]: If it suits the needs of this use case, how about swapping this out for `abbreviateAddress` in `src/lib/utils/address.ts`? Otherwise, this is fine

File: src/components/L2/Dropdowns/Dropdown.tsx
Comment (tomholford) [inline src/components/L2/Dropdowns/Dropdown.tsx:8]: Per [this reference](https://github.com/typescript-cheatsheets/react#useful-react-prop-type-examples) for React + Typescript, it's recommended to use `React.ReactNode` for `children` type declarations for improved type-checking

File: src/components/L2/Dropdowns/FeeDropdown.tsx
Comment (tomholford) [inline src/components/L2/Dropdowns/FeeDropdown.tsx:32]: Are these the values we will be using in production? Or is there an API where we can load updated gas prices?

File: src/components/L2/Headers/L2BackHeader.tsx
Comment (tomholford) [inline src/components/L2/Headers/L2BackHeader.tsx:37]: Let's either log this only in development / tmp environment, or remove completely

File: src/components/L2/Headers/L2PointHeader.tsx
Comment (tomholford) [inline src/components/L2/Headers/L2PointHeader.tsx:28]: Same comment as in `L2BackHeader` re: logging debug info

File: src/components/L2/Window/BodyPane.tsx
Comment (tomholford) [inline src/components/L2/Window/BodyPane.tsx:7]: Looks like this type got lost in the shuffle

File: src/lib/useInviter.js
Comment (tomholford) [inline src/lib/useInviter.js:197]: Has this TODO been addressed?

File: src/lib/useRoller.ts
Comment (tomholford) [inline src/lib/useRoller.ts:146]: Has this TODO been resolved?

File: src/views/IssueChild.js
Comment (tomholford) [inline src/views/IssueChild.js:46]:
```suggestion
```

File: src/lib/useRoller.ts
Comment (tomholford) [inline src/lib/useRoller.ts:293]: These log statements seem useful for debugging, perhaps we should keep these behind a flag for future development?

File: src/views/Invite/Cohort.js
Comment (tomholford) [inline src/views/Invite/Cohort.js:1]: This file you inherited is rather large, is it possible to refactor some of the conditional branches (e.g., the `hasPending` or `networkKeysNotSet` paths) and / or the component itself into smaller submodules? Would make this much more manageable for future reviewers and maintainers. Will defer to you on whether or not to address this now, it might be more trouble than its worth.

File: src/views/TransactionHistory.scss
Comment (tomholford) [inline src/views/TransactionHistory.scss:3]: Is this still needed?

File: src/views/Points.js
Comment (tomholford) [inline src/views/Points.js:151]: Has this TODO been resolved?

File: src/views/Points.js
Comment (tomholford) [inline src/views/Points.js:70]: Please move these to `constants`

File: src/views/Points.js
Comment (tomholford) [inline src/views/Points.js:380]: Is this still needed?

File: src/views/Point.js
Comment (tomholford) [inline src/views/Point.js:302]: Is this still needed?

File: src/views/Point.js
Comment (tomholford) [inline src/views/Point.js:275]: Still needed?

**Review by tomholford — CHANGES_REQUESTED** [review]
Nice work, Will, overall this looks good. Added a bunch of comments and questions in-line. Most are relatively minor, but there's at least one big question: are we going to use hardcoded gas prices in production?

Also, did not call this out inline since it's more of a general observation: in most of the view templates, the `Box` and `Row` components from `@tlon/indigo-react` are used, per the directive from Tlon's design team. However, there are a handful of instances with `div` instead of one of the aforementioned components. My understanding is we should prefer Box or Row when possible to maintain consistent stylings with the rest of Tlon & Urbit's design language.

File: src/components/L2/Card.tsx
Comment (willbach) [reply src/components/L2/Card.tsx:29]: I thought React had a problem with non-boolean values for conditional rendering

File: src/components/L2/Dropdowns/FeeDropdown.tsx
Comment (willbach) [reply src/components/L2/Dropdowns/FeeDropdown.tsx:32]: Right now I'm not using this component. I can remove these but generally yes, they should be coming from an API

File: src/lib/useInviter.js
Comment (willbach) [reply src/lib/useInviter.js:197]: yes, we don't actually use that anymore so it has been removed

File: src/views/Invite/Cohort.js
Comment (willbach) [reply src/views/Invite/Cohort.js:1]: I agree. I might be able to do it tonight.

File: src/views/TransactionHistory.scss
Comment (willbach) [reply src/views/TransactionHistory.scss:3]: yes, but it's still half-finished.

File: src/views/Points.js
Comment (willbach) [reply src/views/Points.js:151]: yes

File: src/views/Points.js
Comment (willbach) [reply src/views/Points.js:380]: nope

File: src/views/Point.js
Comment (willbach) [reply src/views/Point.js:302]: nope

**Comment by willbach** [comment]
@tomholford I'm going to remove the `FeeDropdown` component with the hardcoded gas prices because we won't be using it for awhile and it'll probably look different anyway. Currently gas prices for the `InlineEthereumTransaction` are coming from somewhere else and that works fine for now.

I'll replace `div` with `Box` wherever I'm using it. Otherwise I think I addressed everything going through it just now. New commit(s) incoming.

File: src/views/TransactionHistory.scss
Comment (tomholford) [reply src/views/TransactionHistory.scss:3]: Okay NP

**Review by tomholford — APPROVED** [review]
LGTM, checked it out and verified locally. Nice work 💪

**Pull request merged**
